### PR TITLE
Kafka double cache producer/consumer test is failing - fixed settings…

### PR DESCRIPTION
**Pull request by lukasz-bajor-pl**
… - should work ok now

Fixed some settings really - all worked for me now (even after removing thread.sleeps :) )

### CODE REVIEW

File: core-observable/src/test/java/org/mache/integrations/eventing/TestEventingBase.java [hunk 1]
```diff
@@ -101,11 +102,11 @@ public void consumerIgnoresMessagePublishedForDifferentEntity() throws Interrupt
 
         CoordinationEntryEvent<String> event = new CoordinationEntryEvent<String>(getUuid(), TestOtherEntity.class.getName(),"ID1", EventType.CREATED, new UUIDUtils());
 
-        while(collector.pollWithTimeout(10)!=null);//drain queues
+        //while(collector.pollWithTimeout(10)!=null);//drain queues
```
Comment (jbowkett) [inline core-observable/src/test/java/org/mache/integrations/eventing/TestEventingBase.java]: Commented out code

File: core-observable/src/test/java/org/mache/integrations/eventing/TestEventingBase.java [hunk 2]
```diff
@@ -129,13 +130,13 @@ public void multipleConsumersGetACopyOfPublishedEvent() throws InterruptedExcept
...
-        while(collector1.pollWithTimeout(10)!=null);//d0rain queues
+        while(collector1.pollWithTimeout(120)!=null);//drain queues
 
         consumer2.registerEventListener(collector2);
         consumer2.beginSubscriptionThread();
-        while(collector2.pollWithTimeout(10)!=null);//drain queues
+        while(collector2.pollWithTimeout(120)!=null);//drain queues
 
-        Thread.sleep(3000);
+        //Thread.sleep(3000);
```
Comment (jbowkett) [inline core-observable/src/test/java/org/mache/integrations/eventing/TestEventingBase.java]: Commented out code

**Comment by jbowkett** [comment]
@lukasz-bajor-pl if you can remove the commented-out lines and re-push then I will merge the PR

**Comment by lukasz-bajor-pl** [comment]
Hi @jbowkett - removed comments also added @suppressWarnings annotations to remove raw types warnings

**Pull request merged**